testData/preferences/MikelVargas.json

```diff
@@ -1,5 +1,6 @@
 {
     "flat": {
+        "name": "MikelVargas",
```

Comment:
Should be "Mikel Vargas"

Reply:
... and yes, I realize that you're tempted to call him "Miguel"

Reply:
Done!